framework/src/task_manager/task_manager_internal.h

```diff
@@ -134,6 +140,13 @@ struct tm_pthread_info_s {
 typedef struct tm_pthread_info_s tm_pthread_info_t;
 #endif
 
+struct tm_unicast_internal_s {
```

Comment:
what's the meaning of internal?

Reply:
for checking the unicast type(sync, async), int type is needed, but user will not use type directly, only internally used.